app-directory/README.md

```diff
@@ -0,0 +1,51 @@
+# Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
```

Comment:
This should also go, this has been replaced by the equivalent file in the doc folder.

If there are README files in the src folder, they should be about how to consume and contribute to the source code.

Reply:
removed in latest commit